samples/04_gis_analysts_data_scientists/detecting_and_categorizing_brick_kilns_from_satellite_imagery.ipynb

```diff
@@ -153,10 +153,14 @@
   },
```

Comment:
<p>should the connection be 'home'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/789/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>